backend/src/ml_space_lambda/app_configuration/lambda_functions.py

```diff
@@ -0,0 +1,64 @@
+#
+#   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License").
+#   You may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import json
+import logging
+import time
+
+from ml_space_lambda.data_access_objects.app_configuration import AppConfigurationDAO, AppConfigurationModel, SettingsModel
+from ml_space_lambda.utils.common_functions import api_wrapper
+
+log = logging.getLogger(__name__)
+app_configuration_dao = AppConfigurationDAO()
+
+
+@api_wrapper
+def update_configuration(event, context):
+    request = json.loads(event["body"])
+    configScope = request["configScope"]
+    version_id = request["versionId"] + 1  # increment so this will be the latest version
```

Comment:
I wanted to increment before we instantiate the AppCofnigurationModel so that the latest version ID is always in the object, as opposed to incrementing it sometime later.

Reply:
Fair enough, it's just personal preference thing and I wanted to throw out the idea.